src/microceph.py

```diff
@@ -83,6 +83,38 @@ def is_cluster_member(hostname: str) -> bool:
             raise e
 
 
+def bootstrap_cluster(params: dict):
+    """Bootstrap MicroCeph cluster."""
+    micro_ip = params.get("micro_ip", None)
+    public_net = params.get("public_net", None)
+    cluster_net = params.get("cluster_net", None)
+
+    cmd = ["microceph", "cluster", "bootstrap"]
+
+    if public_net:
+        cmd.extend(["--public-network", public_net])
+
+    if cluster_net:
+        cmd.extend(["--cluster-network", cluster_net])
+
+    if micro_ip:
+        cmd.extend(["--microceph-ip", micro_ip])
+
+    _run_cmd(cmd=cmd)
+
+
+def join_cluster(params: dict, token: str):
+    """Join node to MicroCeph cluster."""
+    micro_ip = params.get("micro_ip", None)
```

Comment:
Why not use keyword args?

```
def join_cluster(token: str, micro_ip: str = None):
```

Reply:
ack, I have also added `**kwargs` so that we can dump the same dict expansion into this method as well.